server/routes/offenceCodeDecisions/prisonerOutsideEstablishmentDecisionHelper.ts

```diff
@@ -0,0 +1,29 @@
+// All functionality that knows about the prisoner-outside-establishment decision.
+import { Request } from 'express'
+import { DecisionForm } from './decisionForm'
+import DecisionHelper from './decisionHelper'
+import { FormError } from '../../@types/template'
+import { OffenceData } from './offenceData'
+import DecisionTreeService from '../../services/decisionTreeService'
+
+export default class PrisonerOutsideEstablishmentDecisionHelper extends DecisionHelper {
```

Comment:
This is a null implementation - it needs to be filled out as part of the UI work